project/QuentinDucoulombier/README.md

```diff
@@ -0,0 +1,109 @@
+# (Project Proposal) DeliveryOptimizer: A Simplified Home-to-Home Delivery Optimization Tool
+
+## Basic Information
+
+DeliveryOptimizer aims to provide a simplified yet efficient solution for
```

Comment:
Traveling salesman problem is difficult to make a good presentation because there are few known parts for you to show how to deal with insufficient knowledge to the problems through your software architecture.

It's not wrong to try, but it's risky.